README.md

```diff
@@ -812,7 +810,8 @@ a server side client.
    another compatible socket.
 
    ```js
-   var Socket = Primus.createSocket({ transformer: transformer, parser: parser })
+   var Primus = require("primus"), //Primus library from npm install primus
```

Comment:
There is an extra comma. Also would you mind adding a space after `//` and use single quotes for consistency?